### ISSUE TITLE
Accessible Navigation Markup

### ISSUE THREAD

**Issue by jmbarne3** [UCF/FinAid-Child-Theme#47]
**Describe the bug**
The right sidebar navigation adds an aria-labelledby to each list item, instead of to the ul.

https://github.com/UCF/FinAid-Child-Theme/blob/4c429c1d12a0eed4c4d7acb1341f8ad3795dd361/includes/right-sidebar-functions.php#L83

**Expected Behavior**
A single aria-labelledby should be applied to the `ul` or `nav` div.

**Possible Fix**
Update the navigation to use the following markup:

```html
<h2 id="right-sidebar-nav" [...]>
<nav aria-labelledby="right-sidebar-nav">
   <ul class="nav flex-column">
      <li class="nav-item"><a href="[...]" class="nav-link"></a></li>
   </ul>
</nav>
```
